index.html

```diff
@@ -207,6 +207,71 @@ <h2 class="headings">Tonics</h2>
 </div>
 
  <div class="card-works">
+  <header class="about-me"> 
+    <h2> About myself</h2>
+  </header>
+   
+
+   <p class="abt-p">
+    Hello I’m a software developer! I can help you build a product , feature or website. 
+    Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+   </p>
+
+   <h2 class="lets-connect">let's connect</h2>
+
+   <div class="link">
+    <ul class="media-icons">
+      <li>
+        <a href="#">
+          <img src="./images/Twitter.png" alt="twitter">
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img src="./images/Linkedin icon.png" alt="linkedin">
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img src="./images/Mail.png" alt="mail">
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img src="./images/Github.png" alt="github">
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img src="./images/Vector.png" alt="vector">
+
+  
+
+  </div>
+    <h1 class="resume">Get my resume</h1>
+  
+     <div class="langs">
+      <h1 class="lang1"> languages</h1>
```

Comment:
- [ ] I'll suggest you use `h2 or h3` tag here as you are meant to have only 1 h1 in your webpage and that should be the main title. :+1: :smile: 

- [ ] As part of the requirement is to replicate the given design as much as possible, I think there are some improvements you need to make here. 
- - The text color of the text `languages` is different from the one in the given design, Please look into this.
- - The spacing between `Frameworks` and `Skills` in your design is more than the given design, I'll suggest you get the appropriate spacing and margin from the design and adjust yours accordingly.
- - Also you are missing the arrows at the right of the languages and frame works, I'll suggest you added this too to your design. :+1: :+1: 

- YOUR DESIGN:
![Selection_032](https://user-images.githubusercontent.com/107632780/236298537-8f1d627a-8e31-4ab2-9c34-c5bf87d9be9a.png)

- FIGMA DESIGN:
![Screenshot from 2023-05-04 19-31-32](https://user-images.githubusercontent.com/107632780/236298600-d4866404-4e0e-4e2a-a019-ca23258052b5.png)